tests/Operation/WatchFunctionalTest.php

```diff
@@ -1382,7 +1382,7 @@ public function testOriginalReadPreferenceIsPreservedOnResume(): void
             ChangeStream::class,
         );
         $cursor = $getCursor();
-        assert($cursor instanceof Cursor);
```

Comment:
I'm not sure why this was using `assert()` instead of a PHPUnit assertion. Apart from this, I only see `assert()` used in a couple of spec prose tests and TestFileCodec.

Consider replacing with `self::assertInstanceOf(CursorInterface::class, $cursor)` if you like.